toolium/utils/data_generator.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) Telefónica Digital.
+# CDO QA Team <qacdo@telefonica.com>
```

Comment:
Toolium Apache license header must be used

Reply:
OK, fixed.